ingest/change.go

```diff
@@ -68,6 +68,39 @@ func GetChangesFromLedgerEntryChanges(ledgerEntryChanges xdr.LedgerEntryChanges)
 	return changes
 }
 
+// GetChangesFromLedgerEntryEvictions transforms evicted LedgerKeys to []Change.
+// The generated changes always remove the entries.
+func GetChangesFromLedgerEntryEvictions(keys []xdr.LedgerKey) []Change {
+	changes := []Change{}
+
+	for _, key := range keys {
+		state := xdr.LedgerEntry{}
+		switch key.Type {
+		case xdr.LedgerEntryTypeContractData:
+			state.Data.SetContractData(&xdr.ContractDataEntry{
+				Contract:   key.ContractData.Contract,
+				Key:        key.ContractData.Key,
+				Durability: key.ContractData.Durability,
+			})
+		case xdr.LedgerEntryTypeContractCode:
+			state.Data.SetContractCode(&xdr.ContractCodeEntry{
+				Hash: key.ContractCode.Hash,
+			})
+		default:
+			// Currently only contractData and contractCode are evicted by core, so
+			// we only need to handle those two.
+			panic("Invalid LedgerEntry eviction type")
```

Comment:
```suggestion
			panic(fmt.Errorf("Invalid LedgerEntry eviction type: %v", key.Type))
```

friendlier :) (might need an `import fmt`)